src/core/pyproject.toml

```diff
@@ -27,7 +27,7 @@ dependencies = [
     "python-dotenv",
     "gunicorn",
     "psycogreen",
-    "psycopg2",
+    "psycopg2-binary",
```

Comment:
For production psycopg2 is the recommended package - see: https://pypi.org/project/psycopg2-binary/